src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -223,10 +221,11 @@ export const ContactPolicy = InjectAppServices(
               >
                 {({ values, errors, isSubmitting, isValid, dirty, setFieldValue }) => (
                   <>
-                    <Prompt
+                    {/* TODO: this is not currently supported in the current released version of v6 (https://reactrouter.com/docs/en/v6/upgrading/v5#prompt-is-not-currently-supported) */}
+                    {/* <Prompt
                       when={dirty && !modalIsOpen}
                       message={_('common.unsaved_changes_message')}
-                    />
+                    /> */}
```

Comment:
Que es esto?

Reply:
Eso es para que cuando el usuario recargue la página le aparezca una ventana de confirmación indicando que va a perder todos los cambios. El detalle es que en la v6 no esta actualmente soportado, pero dicen que mas adelante van a desarrollar algo para ese comportamiento que se pierde https://reactrouter.com/docs/en/v6/upgrading/v5#prompt-is-not-currently-supported

Reply:
Creo que eso lo hicimos con JavaScript acá: https://github.com/FromDoppler/doppler-editors-webapp/blob/main/src/components/SingletonEditor.tsx#L110-L118

Sería eso?

Reply:
Creo que ese te lo guarda, el prompt que utiliza alli en contact policy no te guarda nada, solo es para avisar al usuario de que ha modificado el formulario y que si sale de la pagina va a perder los cambios

Reply:
Pareciera que `<Prompt>` si tiene que ver con el evento de `beforeunload`: https://github.com/ZacharyRSmith/react-router-navigation-prompt/blob/master/src/index.js#L77

---

Que vas a hacer respecto a este punto? es aceptable que se pierda esa funcionalidad?

Reply:
Si, es el mismo evento. Pero, creo que antes de implementar esa funcionalidad habría que definir si otros formularios deben tener el mismo comportamiento. Me parece un poco raro que ese comportamiento se encuentre solo en Contact Policy. Que opinas?

Reply:
Ah, claro, en ningún otra parte de Doppler es así, no me parece mal que se quite esa feature.

Gracias por la aclaración!